aries_cloudagent/ledger/routes.py

```diff
@@ -590,20 +594,35 @@ async def ledger_accept_taa(request: web.BaseRequest):
             raise web.HTTPForbidden(reason=reason)
 
     accept_input = await request.json()
+    LOGGER.info(">>> accepting TAA with: %s", accept_input)
     async with ledger:
         try:
             taa_info = await ledger.get_txn_author_agreement()
             if not taa_info["taa_required"]:
                 raise web.HTTPBadRequest(
                     reason=f"Ledger {ledger.pool_name} TAA not available"
                 )
+            LOGGER.info(f"TAA on ledger: {taa_info}")
+            # this is a bit of a hack, but the "\ufeff" code is included in the
+            # ledger TAA and digest calculation, so it needs to be included in the
+            # TAA text that the user is accepting
+            # (if you copy the TAA text using swagger it won't include this character)
+            if taa_info["taa_record"]["text"].startswith("\ufeff"):
+                if not accept_input["text"].startswith("\ufeff"):
+                    LOGGER.info(
+                        ">>> pre-pending -endian character to TAA acceptance text"
+                    )
+                    accept_input["text"] = "\ufeff" + accept_input["text"]
```

Comment:
I ran into this issue specifically on the Sovrin builder net, not sure if it would happen on the other networks.